app/serializers/basic_post_serializer.rb

```diff
@@ -45,7 +45,7 @@ def cooked
 
   def ignored
     return false unless SiteSetting.ignore_user_enabled?
-    object.is_first_post? &&
+    (object.is_first_post? || object.reply_to_post_number.present?) &&
```

Comment:
Since this is now potentially called for more than 1 posts per topic, can we somehow prevent the N+1?

Reply:
Good point. Let me try and see if we can just do it all front-end.

Reply:
Found a way out by delegating hiding the content work to the front-end. ✅